sway/sway.5.scd

```diff
@@ -725,12 +725,14 @@ The default colors are:
 *unbindswitch* <switch>:<state>
 	Removes a binding for when <switch> changes to <state>.
 
-*unbindsym* [--whole-window] [--border] [--exclude-titlebar] [--release] [--input-device=<device>] <key combo>
+*unbindsym* [--whole-window] [--border] [--exclude-titlebar] [--release] [--locked] \
+		[--to-code] [--input-device=<device>] <key combo>
```

Comment:
We need strip leading whitespace here too, I believe.

Reply:
Yeah looks like it. The specific width of my terminal hid this bit of formatting from me in the man page.